src/exabgp/reactor/api/command/neighbor.py

```diff
@@ -73,99 +63,98 @@ class Neighbor(object):
     summary_template = '%-15s %-7s %9s %-12s %10d %10d'
 
     @classmethod
-    def as_dict(cls, answer):
```

Comment:
as_dict should not be inlined. It would be better to have as _string added and use two functions for clarity.